spyder/plugins/variableexplorer/widgets/tests/test_arrayeditor.py

```diff
@@ -70,6 +76,16 @@ def test_object_arrays_display(qtbot):
     assert u'[Numpy array]' == dlg.arraywidget.model.data(idx)
 
 
+def test_attribute_errors(qtbot):
+    """
+    Verify that we don't get a AttributeError for certain structured arrays.
+
+    Fixes spyder-ide/spyder#11216 .
+    """
+    from scipy.io import loadmat
+    data = loadmat(os.path.join(FILES_PATH, 'issue_11216.mat'))
+    dlg = setup_arrayeditor(qtbot, data['S'])
```

Comment:
Please `assert` something after this line, i.e. that the array editor contains some of the contents present in `issue_11216.mat`